packages/plugin-core/src/commands/MoveSelectionToCommand.ts

```diff
@@ -112,8 +112,12 @@ export class MoveSelectionToCommand extends BasicCommand<
 
   private createLookupProvider(opts: { activeNote: NoteProps | undefined }) {
     const { activeNote } = opts;
-    return this.extension.noteLookupProviderFactory.create(this.key, {
+    // the id here should be "lookup" as long as we are supplying this
+    // to the lookup command.
+    // TODO: give it its own id once we refactor.
```

Comment:
After refactoring, I don't think we need any ID at all - anyway, irrelevant for now.